tools/changeDelayTimelock/.gitignore

```diff
@@ -0,0 +1,2 @@
+change_delay_output.json
```

Comment:
Add timestamp to the output, so it is not overwritten it the tool is launched several times.
example: https://github.com/0xPolygonHermez/zkevm-contracts/blob/v9.0.0-rc.3-pp/tools/addRollupType/addRollupType.ts#L15